docs/admin/limitrange/README.md

```diff
@@ -86,30 +86,39 @@ Let's create a simple limit in our namespace.
 
 ```console
 $ kubectl create -f docs/admin/limitrange/limits.yaml --namespace=limit-example
-limitranges/mylimits
+limitrange "mylimits" created
 ```
 
 Let's describe the limits that we have imposed in our namespace.
 
 ```console
 $ kubectl describe limits mylimits --namespace=limit-example
-Name:   mylimits
-Type      Resource  Min  Max Default
-----      --------  ---  --- ---
-Pod       memory    6Mi  1Gi -
-Pod       cpu       250m   2 -
-Container memory    6Mi  1Gi 100Mi
-Container cpu       250m   2 250m
+Name:		mylimits
+Namespace:	limit-example
+Type		Resource	Min	Max	Request	Limit	Limit/Request
+----		--------	---	---	-------	-----	-------------
+Pod		cpu		200m	2	500m	1	-
+Pod		memory		6Mi	1Gi	100Mi	1Gi	-
+Container	cpu		100m	2	200m	300m	-
+Container	memory		3Mi	1Gi	100Mi	200Mi	-
 ```
 
 In this scenario, we have said the following:
 
-1. The total memory usage of a pod across all of its container must fall between 6Mi and 1Gi.
-2. The total cpu usage of a pod across all of its containers must fall between 250m and 2 cores.
-3. A container in a pod may consume between 6Mi and 1Gi of memory.  If the container does not
-specify an explicit resource limit, each container in a pod will get 100Mi of memory.
-4. A container in a pod may consume between 250m and 2 cores of cpu.  If the container does
-not specify an explicit resource limit, each container in a pod will get 250m of cpu.
+1. The total memory usage of a pod across all of its container(s) must fall between 6Mi and 1Gi.
```

Comment:
see above comment that default request and default limit are not supported for pod limit type.

If a max constraint is specified for a resource, then a limit must be specified for that resource across all containers.  Failure to specify a limit will result in a validation error when attempting to create the pod.  

If a min constraint is specified for a resource, then a request must be specified for that resource across all containers.  Failure to specify a request will result in a validation error when attempting to create the pod.

So for this table entry, the sum of all containers memory requests must be >= 6Mi and the sum of all containers memory limits must be <= 1Gi.